pkg/upgrade/upgrade.go

```diff
@@ -351,6 +351,46 @@ func UpdateFromLegacyVersion(cli client.Client, platform deploy.Platform, appNS
 	return nil
 }
 
+// CleanupDeprecatedFeatures deletes deprecated features by removing the corresponding FeatureTrackers
+func CleanupDeprecatedFeatures(ctx context.Context, cli client.Client, platform deploy.Platform) error {
+	if platform != deploy.ManagedRhods {
+		return nil
+	}
+
+	crd := &apiextv1.CustomResourceDefinition{}
+	if err := cli.Get(ctx, client.ObjectKey{Name: "featuretrackers.features.opendatahub.io"}, crd); err != nil {
+		return err
+	}
+
+	type ResourceToDelete struct {
+		InstanceName     string
+		GroupVersionKind schema.GroupVersionKind
+	}
+
+	toDelete := []string{
+		"opendatahub-kserve-temporary-fixes", // Workaround for RHOAIENG-165
```

Comment:
@bartoszmajsak would the `opendatahub-` prefix change? is there a const for it?

Reply:
Fixed so that it uses `dscApplicationsNamespace`